blocks/cards/cards.js

```diff
@@ -2,6 +2,7 @@ import { createOptimizedPicture } from '../../scripts/aem.js';
 import { moveInstrumentation } from '../../scripts/scripts.js';
 
 export default function decorate(block) {
+  if (block.classList.contains('hero') && block.classList.contains('cards')) block.classList.remove('cards');
```

Comment:
As cards is the main block name, removing 'cards' class name will create problem.